src/app/components/Paragraph/index.tsx

```diff
@@ -1,7 +1,7 @@
-/** @jsx jsx */
+/** @jsxImportSource @emotion/react */
```

Comment:
Required for emotion to work, can't remember why at the moment, it may be wise to take this away again to understand better why this change is needed